waku/v2/protocol/waku_noise/noise_utils.nim

```diff
@@ -33,6 +33,29 @@ proc randomSeqByte*(rng: var BrHmacDrbgContext, size: int): seq[byte] =
   brHmacDrbgGenerate(rng, output)
   return output
 
+# Pads a payload according to PKCS#7 as per RFC 5652 https://datatracker.ietf.org/doc/html/rfc5652#section-6.3
+proc pkcs7_pad*(payload: seq[byte], paddingSize: int): seq[byte] =
```

Comment:
How about setting the default value of the `paddingSize` parameter to the `PaddingBlockSize`?